.github/workflows/qa.yml

```diff
@@ -106,66 +102,38 @@ jobs:
         if: matrix.os == 'ubuntu-latest' && github.event.pull_request.draft == false
 
       - name: Code linting [Go]
-        if: matrix.project != 'ui' && matrix.project != 'new-ui' && steps.filter.outputs.go == 'true' && github.event.pull_request.draft == false
+        if: matrix.project != 'ui' && steps.filter.outputs.go == 'true' && github.event.pull_request.draft == false
         uses: golangci/golangci-lint-action@v3
         with:
           working-directory: ${{ matrix.project }}
           version: v1.42.1
           args: ${{ matrix.lint_args }}
           skip-pkg-cache: true
 
-      - name: Set up Node.JS 16.13.1 [UI]
-        if: matrix.project == 'ui' && matrix.project != 'new-ui' && steps.filter.outputs.ui == 'true' && github.event.pull_request.draft == false
-        uses: actions/setup-node@v1
-        with:
-          node-version: "16.13.1"
-
-      - name: Set up Node.JS 17.9.1 [NEW-UI]
-        if: matrix.project == 'new-ui' && matrix.project != 'ui' && steps.filter.outputs.ui == 'true' && github.event.pull_request.draft == false
+      - name: Set up Node.JS 17.9.1 [UI]
+        if: matrix.project == 'ui' && steps.filter.outputs.ui == 'true' && github.event.pull_request.draft == false
         uses: actions/setup-node@v1
         with:
           node-version: "17.9.1"
 
       - name: Cache node modules [UI]
         if: matrix.project == 'ui' && steps.filter.outputs.ui == 'true' && github.event.pull_request.draft == false
-        uses: actions/cache@v3
+        uses: actions/cache@v1
```

Comment:
v3

Reply:
I'm just keeping what was already working to avoid any issues for now. The update can be done later